suite-native/module-settings/src/components/FAQInfoPanel.tsx

```diff
@@ -1,56 +1,128 @@
 import { Fragment, ReactNode } from 'react';
 
-import { AccordionItem, Text, VStack } from '@suite-native/atoms';
-import { Link } from '@suite-native/link';
+import { AccordionItem, Box, Text, VStack } from '@suite-native/atoms';
+import { useIsUsbDeviceConnectFeatureEnabled } from '@suite-native/feature-flags';
+import { FlatList } from 'react-native';
 
 type FAQ = {
     question: string;
     answer: ReactNode;
 };
 
-const faqMap: FAQ[] = [
+type BulletListProps = {
+    texts: string[];
+};
+
+const BulletList = ({ texts }: BulletListProps) => {
+    const data = texts.map(text => ({ key: text }));
+    return (
+        <FlatList
+            data={data}
+            renderItem={({ item }) => (
+                <Box flexDirection="row">
+                    <Text variant="label"> • </Text>
+                    <Text variant="label">{item.key}</Text>
+                </Box>
+            )}
+        />
+    );
+};
+
+const faqMapNoUsb: FAQ[] = [
```

Comment:
please move these strings to translation file

Reply:
d091552781